### ISSUE TITLE
Understanding Comet code

### ISSUE THREAD

**Issue by joannacknight** [alan-turing-institute/ARC-MTQE#20]
We need a better understanding of some aspects of the COMET code base. 

A few things to consider:

- Relationship between the hyperparameters `encoder_model`, `pretrained_model` and `load_pretrained_weights`
- Is the `loss` argument actually used for anything? Seems that the sentence loss is hard-coded in the `init_losses` method
- Unfreezing the encoder - want to make sure we are doing this correctly.
- Batch size - how to set this correctly

As part of this issue we will work out which hyperparameters we want to search and which we want to be able to set ahead of training.
